README.md

```diff
@@ -59,29 +56,32 @@ This will be the file that holds your S3 and SQS configurations.
     - `FEDERALIST_AWS_BUILD_KEY` is `accessKeyId`
     - `FEDERALIST_SESSION_SECRET` is `secreyAccessKey`
     - `FEDERALIST_SQS_QUEUE` is `queue`
-1. Create Postgres databases by running
-    - Run `createdb federalist`
-    - Run `createdb federalist-test`
-1. Great work! The Federalist app is now ready to run locally! :tada:
-    - Run `yarn`
-    - Run `yarn build`
-    - Run `yarn start`
-1. You should now be able to see Federalist running at http://localhost:1337/
+1. Run `docker-compose build`.
+1. Run `docker-compose run app yarn && docker-compose run app yarn build` to install dependencies and build the app initially.
+1. Run `docker-compose up` to start the development environment.  You should now be able to see Federalist running at http://localhost:1337/. Local file changes will cause the server to restart and/or the front end bundles to be rebuilt.
 
 **Pro tips:**
-- You can use `yarn watch` to automatically restart the server and rebuild front end assets on file change, which is useful for development.
-- You can use `yarn test` to run local testing on the app.
 
+In our Docker Compose environment, `app` is the name of the container where the Federalist web application runs. You can run any command in the context of the web application by running  `docker-compose run app <THE COMMAND>`.
+
+For example:
+
+- Use `docker-compose run --service-ports app yarn test` to run local testing on the app.
+- Use `docker-compose run app yarn lint:diff` to check that your local changes meet our linting standards.
+
+Similarly you can run any command in the context of the database container `db` by running `docker-compose run db <THE COMMAND>`.
+
+Note that when using `docker-compose run`, the docker network will not be exposed to your local machine. If you do need the network available, run `docker-compose run --service-ports app <THE COMMAND>`.
 
 #### Build the server and the front-end
 
-If you are working on the front-end of the application, the things you need to know are:
+If you are working on the front end of the application, the things you need to know are:
 
-1. It is a React based application
+1. It is a React and Redux application
 1. It is built with `webpack`
 1. It lives in `/frontend`
 
-To analyze the contents of the front end JavaScript bundle, use `yarn analyze-webpack` after a build. This will launch a browser window showing a visualization of all the code that makes up the bundle.
+To analyze the contents of the front end JavaScript bundle, use `docker-compose run --service-ports app yarn analyze-webpack` after a build. Then visit http://127.0.0.1:8888 to see a visualization of the the bundle contents.
```

Comment:
This doesn't work yet. I think it is because I need to change the args to `webpack-bundle-analyzer` to use localhost instead of the 127.0.0.1 IP.

Reply:
works now -- had to set it to use `0.0.0.0`